### DISCUSSION TITLE
Assistance Needed for Running Multiple NetBox Instances on Docker

### DISCUSSION THREAD

**Discussion by dongepi** [netbox-community/netbox-docker#1319] [Getting Help]
Hi everyone,

I’m struggling to launch a second instance of NetBox on the same Docker machine. I’ve already changed all the volume names and made other adjustments that I could think of, but each attempt results in the instance crashing with various errors.

Has anyone successfully managed to run more than one NetBox instance on the same Docker setup? Any insights or guidance would be greatly appreciated.

Thanks in advance!

**Comment by aascedu** [comment]
Make sure to change the port and the container names !
